src/convertToObject.js

```diff
@@ -6,7 +6,32 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const result = {};
+
+  // Split the input string by semicolon to separate each CSS declaration
+  const declarations = sourceString.split(';');
+
+  for (const declaration of declarations) {
+    // Ignore empty declarations
+    if (!declaration.trim()) {
+      continue;
+    }
+
+    // Split each declaration by colon to separate the property and value
+    const [property, value] = declaration.split(':');
+
+    if (property && value) {
```

Comment:
The current check `if (property && value)` will fail if the value is an empty string or contains only whitespace. Consider trimming both `property` and `value` before this check to ensure that only meaningful pairs are processed.